src/sdk/pynni/nni/curvefitting_assessor/curvefitting_assessor.py

```diff
@@ -51,9 +50,7 @@ def __init__(self, epoch_num=20, optimize_mode='maximize', start_step=6, thresho
         logger.info('Successfully initials the curvefitting assessor')
 
     def trial_end(self, trial_job_id, success):
-        '''
-        trial end: update the best performance of completed trial job
-        '''
+        """trial end: update the best performance of completed trial job"""
```

Comment:
seems we need to point out the type of the parameters?

Reply:
Add already, please help review again :)